### PR TITLE
Add segments = "both"

### PR DESCRIPTION
**Pull request by mem48**
Option to return both routes and segments as a list of two data frames rather than either-or choice

See https://github.com/nptscot/npt/issues/243

### CODE REVIEW

**Comment by Robinlovelace** [comment]
:fire:

File: R/batch.R [hunk 1]
```diff
@@ -196,7 +198,8 @@ batch = function(
 
 get_routes = function(url, desire_lines = NULL, filename, directory,
                       cols_to_keep = c("id", "name", "provisionName", "distances", "time",
-                                       "quietness", "gradient_smooth")) {
+                                       "quietness", "gradient_smooth"),
+                      segments = TRUE) {
```
Comment (Robinlovelace) [inline R/batch.R:202]: Sensible default.

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Please add some examples under `@examples`.

**Comment by mem48** [comment]
I think this is now working  I had to make this change to get it working which I think is a bug in the current version

https://github.com/cyclestreets/cyclestreets-r/pull/80/commits/6110b3e672e44d34353968c3311fffa0801146fd

**Pull request merged**